app/pages/OrganizationListingPage.tsx

```diff
@@ -60,9 +58,6 @@ export const OrganizationListingPage = () => {
   );
```

Comment:
On the lines just above this, could you remove the `"feedback"` string from both arrays? I can't comment directly on the lines, since they're just beyond the range of what the GitHub interface allows me to comment on, but they're just above this line.